### ISSUE TITLE
[Design] - Update Native Share Mocks

### ISSUE THREAD

**Issue by grgia** [Expensify/App#42395]
See https://docs.google.com/document/d/14C1VifxvIXeyLAf0XstbC-wvyAb5SQZ0AOw0-jLRyZs/edit?disco=AAABNPHRAeY

**Comment by grgia** [comment]
@dannymcclain there's some comments throughout the doc, but the mocks feature some outdated icons and components, could you help me ensure that all the docs are up to date?

cc @dubielzyk-expensify as you worked on the original mocks

**Comment by dubielzyk-expensify** [comment]
Let me know if you got any Qs, Danny! 😄

**Comment by dannymcclain** [comment]
Left a few questions and got some input from Jon, but I think these are pretty much there.

[Figma link](https://www.figma.com/design/qZ8Ynbx2XZOlQC4prN9Nwc/Native-Share-Menu?node-id=401-4875&t=EK9mbnqxBhoyLhRn-11)

**Comment by dannymcclain** [comment]
Ok I think the mocks are essentially done, just need to iron out if we're going to include Split in this release, and make any necessary tweaks based on that decision (they'd be minor and take 2 seconds). [Little convo about including Split here in Slack.](https://expensify.slack.com/archives/C036QM0SLJK/p1716468164130469)

How are we going to make that decision? I feel like this is kinda held up on that.

**Comment by dannymcclain** [comment]
🤨 Ok Melvin chill out a bit.

**Comment by grgia** [comment]
@dannymcclain thanks for the shout, I brought this one to slack https://expensify.slack.com/archives/C036QM0SLJK/p1717492905833899

**Comment by dannymcclain** [comment]
Mocks are all up-to-date in Figma. I noted that Split is out of scope right now.

https://www.figma.com/design/qZ8Ynbx2XZOlQC4prN9Nwc/Native-Share-Menu?node-id=401-4875&t=gAU4PRYaS2dh7AtF-11

Low-res screenshot previews below:

### Share
![CleanShot 2024-06-04 at 14 23 50@2x](https://github.com/Expensify/App/assets/7516624/87369173-f6b2-4b0e-9f54-45fba75917b2)

### Submit
![CleanShot 2024-06-04 at 14 24 05@2x](https://github.com/Expensify/App/assets/7516624/8aae5d2f-7b02-45d4-a547-0163e5a49d52)

### Errors
![CleanShot 2024-06-04 at 14 24 19@2x](https://github.com/Expensify/App/assets/7516624/71a953ed-04e7-41f5-a5bc-1fbe9aab5681)

**Comment by dannymcclain** [comment]
I think we might even be good to close this unless you want some help updating the mocks in the design doc?

**Comment by dannymcclain** [comment]
[Same as above.](https://github.com/Expensify/App/issues/42395#issuecomment-2155435846)

**Comment by dannymcclain** [comment]
Yo @grgia I'm going to go ahead and close this since the mocks are done. Reopen it or hit me up on Slack/ND if you need anything else!
